wavelength/settings.py

```diff
@@ -32,8 +32,8 @@
 DEBUG = int(os.environ.get('DEBUG', default=0))
 
 # validations
-ALLOWED_HOSTS = ['localhost, 127.0.0.1, vast-earth-36970.herokuapp.com']
-CSRF_TRUSTED_ORIGINS = 'https://vast-earth-36970.herokuapp.com'
+ALLOWED_HOSTS = ['*'] #['localhost, 127.0.0.1 , vast-earth-36970.herokuapp.com']
```

Comment:
we will temporarly allow this, in the future we will secure this better.  Nothing need to be done